src/lib/functions/background.js

```diff
@@ -1,12 +1,16 @@
-const lambdaLocal = require('lambda-local')
-
 const { NETLIFYDEVERR, NETLIFYDEVLOG } = require('../../utils/logo')
 
-const { DEFAULT_LAMBDA_OPTIONS, formatLambdaError, SECONDS_TO_MILLISECONDS, styleFunctionName } = require('./utils')
+const { formatLambdaError, styleFunctionName } = require('./utils')
 
 const BACKGROUND_FUNCTION_STATUS_CODE = 202
 
-const createBackgroundFunctionCallback = (functionName) => (err) => {
+const handleBackgroundFunction = (functionName, response) => {
```

Comment:
We remove `lambda-local` since it's specific to the JavaScript runtime. Instead, this file simply receives the name of a function and the response object, being responsible for terminating the response.